packages/@react-aria/i18n/docs/internationalization.mdx

```diff
@@ -0,0 +1,146 @@
+logographic scripts (e.g. Chinese and Japanese) are written left to right. Other languages such as Arabic and Hebrew are written
+right to left. These languages are called “bi-directional,” or are also commonly referred to as “RTL” (“right-to-left”) languages.
+
+In right-to-left languages, user interfaces are expected to be **mirrored**. Layouts flip such that components are positioned on the
+opposite side of the interface. For example, a button that would be displayed on the right side of a screen in a left-to-right language
+would appear on the left side of the screen in a right-to-left language. In addition, elements within a component are also expected to
+mirror. For example, in a checkbox component, the label would be placed on the left side of the checkmark rather than the right.
+
+Since React Aria provides no rendering, it is up to you to implement RTL support in your design. The documentation for individual react-aria hooks
+calls out when you need to be aware of this, so you can account for it in your CSS using 
+[logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) or other means. For example, flexbox and CSS grid
+both automatically flip their layouts depending on the direction.
+
+Even though it is not responsible for rendering or layout, React Aria is aware of the current directionality, and adjusts interactions 
+accordingly. For example, when navigating using the left and right arrow keys, React Aria automatically flips the direction so that 
+the left arrow always navigates to the item physically to the left, and the right arrow always navigates to the item physically to the right.
+
+## Example
+
+The root most element of your application should define the [lang](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang)
+and [dir](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/dir) attributes so that the browser knows which language
+and direction the user interface should be rendered in. This can be done with the [useLocale](useLocale.html) hook from `@react-aria/i18n`.
+
+```tsx
+import {useLocale} from '@react-aria/i18n';
+
+function YourApp() {
+  let {locale, direction} = useLocale();
+
+  return (
+    <div lang={locale} dir={direction}>
+      {/* your app here */}
+    </div>
+  );
+}
+```
+
+React Aria automatically detects the user's current language by default, and even updates this if the browser or system language changes.
+However, if you would like to override this with an application specific setting, you can do so using the [I18nProvider](I18nProvider.html)
+exposed by `@react-aria/i18n`.
+
+```tsx
+import {I18nProvider} from '@react-aria/i18n';
+
+<I18nProvider locale="fr-FR">
+  <YourApp />
+</I18nProvider>
+```
+
+## Supported locales
+
+React Aria currently supports over 30 locales. They are listed below.
+
+<div style={{columnWidth: 200}}>
+
+* Arabic (United Arab Emirates)
```

Comment:
should we include the formatted name? `en-US` etc

Reply:
hmm, I don't think we need it? We already cover both the language and region here, so the same info is already here.